### ISSUE TITLE
Revert pleading emoji or accept/create a more representative redesign

### ISSUE THREAD

**Issue by Cassumbra** [jdecked/twemoji#32]
The current redesign diverges way too far from the previous version and expresses an entirely different intent. It also shares much too similarity with the current "face holding back tears" design.
I am making this as a separate issue from #12 because I feel as though that issue does not carry the same intent as this one. Apologies if they are too similar.
To clarify:
I see this issue as: "Please change or redesign the pleading emoji"
Whereas #12 seems to be: "Pleading emoji is fine, but don't make any more changes"

Perhaps I am simply misreading the text though. Sorry if so.

**Comment by Woofer21** [comment]
I disagree here, the new pleading emoji design brings it inline with all other emoji designs, which is important to have when someone on one platform could see the emoji as something and someone on a different platform could see it totally differently.

You can see [here](https://emojipedia.org/pleading-face/) how the old one does not fit in at all with any of the others.

**Comment by TomasTokaMrazek** [comment]
I personally dislike the new pleading emoji quite a lot, as it conveys completely different meaning. But I have to agree with @Woofer21.

Your best bet is to add it as custom server emoji.

**Comment by barkloaf** [comment]
there is nothing wrong with the original pleading face design. I'm not saying anything about the sentiment expressed in #12 but what I will say is that I think that the old pleading face should be brought back as soon as possible without the prospect of redesigning it (hence #27). it's not really that different from the other vendors (variation among vendors is to be expected!) and I am struggling to think how it could be misinterpreted. furthermore, due to the cultural impact that the old pleading face had, I think it's pretty likely that even if the emoji is ambiguous (which, once again, I believe it not to be), people will have been exposed to it enough to know which emoji it's supposed to be. the original twemoji pleading face is very quintessentially a pleading face; I think it's the other vendors that aren't making sense.

as for the notion of "adding it as a custom server emoji" that @TomasTokaMrazek mentioned.. I assume you're referring to discord? if so, discord requires you to pay for the usage of other servers' emojis when you're not in that server. this is not a solution; why would I want to give my money to discord? furthermore, more platforms than just discord use twemoji; this isn't really about discord at all. any time I have the opportunity to set which emoji set an application uses, I choose twemoji. I even have it as the only emoji set installed on my arch system so that all emojis are forced to be twemoji. I am under the belief that twemoji is the only good emoji set. I believe it should be the definitive design for emojis and that's why I care so much about bringing back a great design.

**Comment by bsoap-cat** [comment]
> there is nothing wrong with the original pleading face design. I'm not saying anything about the sentiment expressed in #12 but what I will say is that I think that the old pleading face should be brought back as soon as possible without the prospect of redesigning it (hence #27). it's not really that different from the other vendors (variation among vendors is to be expected!) and I am struggling to think how it could be misinterpreted. furthermore, due to the cultural impact that the old pleading face had (which is even mentioned on the emojipedia page!), I think it's pretty likely that even if the emoji is ambiguous (which, once again, I believe it not to be), people will have been exposed to it enough to know which emoji it's supposed to be. the original twemoji pleading face is very quintessentially a pleading face; I think it's the other vendors that aren't making sense.
> 
> as for the notion of "adding it as a custom server emoji" that @TomasTokaMrazek mentioned.. I assume you're referring to discord? if so, discord requires you to pay for the usage of other servers' emojis when you're not in that server. this is not a solution; why would I want to give my money to discord? furthermore, more platforms than just discord use twemoji; this isn't really about discord at all. any time I have the opportunity to set which emoji set an application uses, I choose twemoji. I even have it as the only emoji set installed on my arch system so that all emojis are forced to be twemoji. I am under the belief that twemoji is the only good emoji set. I believe it should be the definitive design for emojis and that's why I care so much about bringing back a great design.

While you _do_ have to pay for Discord Nitro to use custom emojis on other servers, you can just send an image of the pleading emoji or any other kind of custom emoji shrunken down to a smaller size.

**Comment by barkloaf** [comment]
> While you _do_ have to pay for Discord Nitro to use custom emojis on other servers, you can just send an image of the pleading emoji or any other kind of custom emoji shrunken down to a smaller size.

I think I made it pretty clear in my message that this literally isn't about discord. what do you do on a platform that you can't upload images to? what do you do on platforms that have explicit formatting for images (for example, not being able to place them in-between characters like you can with emoji) what about when someone else sends the emoji without the image? I highly advise you re-read what I wrote. emoji exists for a reason and are not substituted by pictures (which was ultimately why #26 was closed).

**Comment by TomasTokaMrazek** [comment]
> > While you _do_ have to pay for Discord Nitro to use custom emojis on other servers, you can just send an image of the pleading emoji or any other kind of custom emoji shrunken down to a smaller size.
> 
> I think I made it pretty clear in my message that this literally isn't about discord. what do you do on a platform that you can't upload images to? what do you do on platforms that have explicit formatting for images (for example, not being able to place them in-between characters like you can with emoji)? what about when someone else sends the emoji without the image? I highly advise you re-read what I wrote. emoji exists for a reason and are not substituted by pictures (which was ultimately why #26 was closed).

It's not about Discord per se, but to be brutally honest, the majority do not care for your projects or your personal emote preference. The fact is, that Discord has 150 milion MAU. In a world of open source its quite normal, that the OSS project is closely related to its biggest user and the user in turn makes contributions to the project. For example pfSense makes a lot of contributions to FreeBSD. From the storm that occured after the pleading emoji change, it's apparent, that Discord should contribute to this project or take over by employing the designer and coder behind twemoji.

There is one clear future of this project. If the development gets disassociated from Discord userbase (as we already see in the recent change of pleading emoji), Discord will eventually make its own emote set (by forking twemoji for example) in order to stop alienating its community. And if Discord stops using this project, it will simply die as it will be very niche emote set for very small userbase. Eventually other projects will rather use Discord emote set, if open source of course. I have seen this scenario a few times already.

**Comment by barkloaf** [comment]
> If the development gets disassociated from Discord userbase (as we already see in the recent change of pleading emoji), Discord will eventually make its own emote set (by forking twemoji for example) in order to stop alienating its community. And if Discord stops using this project, it will simply die as it will be very niche emote set for very small userbase. Eventually other projects will rather use Discord emote set, if open source of course. I have seen this scenario a few times already.

yeah, that's understandable. apologies if I was a little harsh previously, I was under the impression that people were trying to make it seem like the pleading face emoji design was only associated with communities on discord (it was also very cherished on twitter who afaik is still using the original repo).

> it's apparent, that Discord should contribute to this project or take over by employing the designer and coder behind twemoji.
I'm not entirely clear on the situation. twitter is in complete shambles right now for obvious reasons, the CDN that provided the original twemoji seems to have shut down, and I don't think any strides will be made at twitter to implement unicode 15. I know that the maintainer of this fork was a twitter employee, but I'm not sure if they worked on twemoji previously (either way, maintaining this is a selfless job; massively appreciated!).

**Comment by 12Me21** [comment]
This article has some relevant information about the popularity of this emoji: (particularly on twitter)
https://blog.emojipedia.org/a-new-king-pleading-face/

**Comment by ticoeteco23gb** [comment]
> I'd suggest moving the pupils upwards, since that feels like an important feature of the original. (I'd really like to move them further, but I think at that point it differs too much from the other platforms' designs)

This design works better for me!

**Comment by wdym-lmao** [comment]
> This design works better for me!

What about my designs at #12?

**Comment by kkiris** [comment]
> What about my designs at #12?

Personally, I like @12Me21's design of U+1F979-U+1F97A more. However, @wdym-lmao also made a design of U+1F979 (which also needs a redesign imo). I have adapted their redesign into a new design that also resembles 12Me21's. 
Thoughts? (left is 12Me21's & right is my new design that would accompany it)

<img src="https://github.com/user-attachments/assets/ba523928-7312-4ddf-ba2a-36e0eed9e1d3" width="75" title="12Me21's U+1F979-U+1F97A"> . <img src="https://github.com/user-attachments/assets/34ef32aa-7d34-480f-abfb-a266065479a2" width="75" title="kk's U+1F979">
